includes/razorpay-webhook.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+require_once __DIR__.'/../razorpay-payments.php';
+
+require_once __DIR__.'/../razorpay-sdk/Razorpay.php';
+use Razorpay\Api\Api;
+
+class RZP_Webhook
+{
+    protected $razorpay;
+    protected $api;
+
+    function __construct()
+    {
+        $this->razorpay = new WC_Razorpay();
+
+        $this->api = $this->razorpay->getRazorpayApiInstance();
+
+        $this->auto_capture_webhook();
```

Comment:
have a public process function instead, and call the appropriate method depending on the event (payment_authorized in our case). Makes it pluggable for subscriptions